lessons-from-hiring-manager-interviews.rst

```diff
@@ -0,0 +1,165 @@
+A few different reasons were given but I actually think `Joel Spolsky had it right`_ when he said
+top developers "barely ever apply for jobs at all. That's because they already have jobs."
+
+This quote isn't necessarily a slight against people who apply actively to jobs.
+Virtually everyone I know in tech started their career applying to a job.
+Rather, the implication here is that as a developer becomes a mid-level developer
+and then becomes a senior developer, they are more likely to rely on their network
+to somewhat passively job seek rather than going through the front door
+and simply applying on a company website.
+
+Reaching people who aren't actively looking is one area where Read the Docs can really help.
+Companies want to reach them when they are just thinking about a new job or perhaps passively seeking.
+Developers who come to our site are actively building software, not job-seeking.
+Many of them are content at their current position.
+For those that aren't, this is a perfect place to showcase great companies
+and help Read the Docs' visitors to find their next job.
+
+.. _Joel Spolsky had it right: https://www.inc.com/magazine/20070501/column-guest.html
+
+
+Recruiters can be worth it
+--------------------------
+
+In each interview, we asked whether the company used recruiters or recruiting related services (eg. Hired, Triplebyte).
+Universally, hiring managers brought up how expensive recruiters are,
+but most of the people we talked to who had actually used recruiters in the past had a positive impression overall.
+Hiring takes a ton of time and typically takes it from senior folks at the company.
+At smaller startups, that's frequently the CEO or CTO.
+Even though the price is high, recruiters who help speed that process along are worth it.
+
+I think the quote that really summed it up was from a VP of engineering at a startup that just raised a Series B.
+He's trying to grow his team by 10 developers this year (about a 50% increase).
+His experience hiring his first couple developers through a 2-person boutique recruiting firm was great and
+he would use them for hiring everyone "if hiring 10 people didn't mean $100k+" in fees alone.
+
+Full disclosure: Triplebyte is an advertiser on Read the Docs.
+
+
+Filtering resumes isn't a real problem
+--------------------------------------
+
+Every single hiring manager we talked to complained about the time they spent filtering resumes from applicants.
+Some choice quotes I heard included "95% of applicants weren't a fit" even after only reviewing the resume
+and a hiring manager who posted to a very popular remote-only job board told me he got "literally one thousand applications".
+While I believe this is a legitimate complaint,
+I think `Aline Lerner said it best`_ when she said "Engineering hiring isn't a filtering problem. It's a sourcing problem."
+Filtering is just a time-consuming task that hiring managers directly deal with.
```

Comment:
This feels a bit dismissive and negative, especially the heading. Perhaps "Sourcing people is the real problem" is a better angle.